internal/fakecgo/libcgo_freebsd.go

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2022 The Ebitengine Authors
+
+package fakecgo
+
+type (
+	pthread_cond_t  [48]byte
```

Comment:
Did you confirm these are the right sizes and macro definitions?

Reply:
PTHREAD_..._INITIALIZERs are NULL in FreeBSD, so I'll change it to that.